crates/bevy_hierarchy/src/hierarchy.rs

```diff
@@ -20,6 +20,20 @@ pub struct DespawnChildrenRecursive {
     pub entity: Entity,
 }
 
+/// Despawns the given entity and all its children recursively without warnings
+#[derive(Debug)]
+pub struct TryDespawnRecursive {
+    /// Target entity
+    pub entity: Entity,
+}
+
+/// Despawns the given entity's children recursively without warnings
+#[derive(Debug)]
+pub struct TryDespawnChildrenRecursive {
+    /// Target entity
+    pub entity: Entity,
+}
+
```

Comment:
We can just add a `warn` field on the original command, more inline with the functions that do the despawning